spec2sdk/models/entities.py

```diff
@@ -238,25 +238,22 @@ def create_model_field_view(field: ModelField) -> ModelFieldView:
         )
 
 
-class OptionalType(PythonType):
-    inner_py_type: PythonType
-
+class NoneType(PythonType):
     @property
     def dependency_types(self) -> Sequence[PythonType]:
-        return (self.inner_py_type,)
+        return ()
 
     @property
     def type_hint(self) -> str:
-        return self.name or f"{self.inner_py_type.type_hint} | None"
+        return self.name or "None"
```

Comment:
My initial thought was that this should be more correct:
```suggestion
        return self.name or "NoneType"
```
But of course nobody writes it this way (c.f. [this post](https://stackoverflow.com/questions/71120350/none-vs-nonetype-for-type-annotation)). So the current way is the best one and we don't need to change it.

Reply:
Interestingly, that `NoneType` was removed in Python 3 at some point, but reintroduced in 3.10. I didn't know that. I agree with you that `NoneType` is better, but for historical reasons `None` was always used for type hinting and as a value https://peps.python.org/pep-0484/#using-none